docs/visual-testing/integrations/storybook.md

```diff
@@ -139,6 +139,7 @@ Below are the environment variables available in the visual-storybook plugin:
 | `SAUCE_BRANCH_NAME`     |          | The branch name or tag you would like to associate this build with. We recommend using your current VCS branch in CI.                                                                                                            |
 | `SAUCE_PROJECT_NAME`    |          | The label / project you would like to associated this build with.                                                                                                                                                                |
 | `SAUCE_VISUAL_BUILD_ID` |          | For advanced users, a custom build ID. Can be used to create builds in advance. This can be used to parallelize tests, shard, or more. <br/> By default, this is not set and we create / finish a build during setup / teardown. |
+| `SAUCE_VISUAL_CUSTOM_ID` |          | For advanced users, a custom build ID in a user-specified format. Can be used to uniquely identify a build and might include other identifiers such as pipeline ID, commit ID etc. Just like `SAUCE_VISUAL_BUILD_ID`, can be used to parallelize tests, shard, or more. <br/> By default, this is not set and we create / finish a build during setup / teardown.  |
```

Comment:
May be good to align it with the descriptions here? https://www.npmjs.com/package/@saucelabs/visual-storybook

I reworded some of it to make it more clear that the `BUILD_ID` is a sauce labs UUID / ID, and the `CUSTOM_ID` is a generic value from a customer.

Reply:
Oh I wasn't aware of that one. Will align them, thanks